Exercise_B/index.css

```diff
@@ -2,3 +2,31 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
```

Comment:
Perfect implementation of the float and clear css properties! Great work

You could improve this code by collating your height, width and margin into a separate box class that you can add to all of your html elements.

Additionally try to stick top one colour convention, this is important when you work in a company.